apps/wallet-mobile/src/components/QRCodeScanner/QRCodeScanner.stories.tsx

```diff
@@ -13,44 +14,49 @@ const Wrapper = () => {
   const [status] = Camera.useCameraPermissions()
   const [publicKeyHex, setPublicKeyHex] = React.useState(null)
   const [path, setPath] = React.useState(null)
+  const [maskEnabled, setMaskEnabled] = React.useState(true)
 
   const handleOnRead = async ({data}) => {
     const parsedData = JSON.parse(data)
+
     setPublicKeyHex(parsedData.publicKeyHex)
     setPath(parsedData.path)
     action('onRead')
-    return Promise.resolve(false)
+
+    return Promise.resolve(true)
   }
 
   return (
     <View style={{justifyContent: 'space-between', flex: 1}}>
-      <QRCodeScanner onRead={handleOnRead} />
+      <QRCodeScanner onRead={handleOnRead} maskEnabled={maskEnabled} />
```

Comment:
I'd suggest `withMask`
```suggestion
      <QRCodeScanner onRead={handleOnRead} maskEnabled />
```